contentcuration/contentcuration/models.py

```diff
@@ -170,13 +214,166 @@ class Meta:
         # Do not allow two nodes with the same name on the same level
         #unique_together = ('parent', 'title')
 
-class ContentLicense(License):
+
+class ContentKind(models.Model):
+    kind = models.CharField(primary_key=True, max_length=200, choices=content_kinds.choices)
+
+    def __str__(self):
+        return self.kind
+
+class FileFormat(models.Model):
+    extension = models.CharField(primary_key=True, max_length=40, choices=extensions.choices)
+    mimetype = models.CharField(max_length=200, blank=True)
```

Comment:
We added this in for the dropzone element because it needs to have it in the "image/jpg" format to know what types to accept